src/components/LmsConfigurations/index.test.jsx

```diff
@@ -55,6 +66,7 @@ describe('<LmsConfigurations /> ', () => {
   it('return Error Page when any LMS fetch request fails with error (not 404)', async () => {
     LmsApiService.fetchMoodleConfig.mockRejectedValue(badMoodleResponse);
     LmsApiService.fetchCanvasConfig.mockRejectedValue(badCanvasResponse);
+    LmsApiService.fetchBlackboardConfig.mockRejectedValue(badBlackboardResponse);
```

Comment:
Same as above.

Reply:
Assuming this is insinuating we should have a separate test for each config returning an error and an error page being generated, I'm going to respectively disagree. It's overkill.

With individual configs, we produce different components based on which config is in state, so testing that each component is rendered correctly there individually I completely understand. However, we are taking ANY 400/>404 and returning an error page component. We don't need to test that functionality 3+ times.